app/models/chargeback_rate_detail.rb

```diff
@@ -91,4 +121,29 @@ def rate_type
     # Return parent's rate type
     chargeback_rate.rate_type unless chargeback_rate.nil?
   end
+
+  # Check that tiers are complete and disjoint
+  def complete_tiers
+    cbts = chargeback_tiers
+    start = -Float::INFINITY
+    tier_ends = false
+    error = false
+    unless cbts.nil?
+      cbts.each do |tier|
+        if tier.start != start
+          errors.add(:chargeback_tiers, "must include all values from -∞ to ∞ once.")
```

Comment:
I don't see a test for this.  What situations would this be in error and when is it ok?